autoload/lsp/completion.vim

```diff
@@ -593,6 +592,7 @@ def LspCompleteDone(bnr: number)
     completionData = lspserver.resolveCompletion(completionData, true)
   endif
   if !completionData->get('additionalTextEdits', {})->empty()
+	\ && opt.lspOptions.completionTextEdit
```

Comment:
Moving the line here seems a bit fishy to me for another reason:

Suppose that `completionTextEdit == false`, then only guarding this if condition may perform redundant work. The delayed `lspserver.resolveCompletion` directly above this if-block might still be executed, but this is completely unnecessary as we won't use the result anywhere.

Reply:
well, simplest change i had said in https://github.com/yegappan/lsp/pull/389#issuecomment-1705627325 (but ignore that cursor issue, you had refined it), if just added and moved 'command' part before 'textedit', then this PR no required and nothing needed change;
i am still not sure what case required such 'command' action at here, could you verify your faced case to verify if it's ok to move that part to there (of course please be sure 'command' if was really required)?

Reply:
> if just added and moved 'command' part before 'textedit', then nothing needed change

I already told you in my original PR that this is not possible, the command has to be applied after the textedit.

> if it's ok to move that part to there

With the latest commit your PR seems fine to me, although I am not an expert with snippet plugins, as I don't use any.

Reply:
> > if just added and moved 'command' part before 'textedit', then nothing needed change
> 
> I already told you in my original PR that this is not possible, the command has to be applied after the textedit.

yes, but i hear `I am not entirely sure` in that comment.

or now anyway seems only 'haskell' (as you said) worked like this, i have no idea any others, and seems you confirmed it should work like this way, then i think this PR is ready to go.